### PR TITLE
📖 Update example1.md to reflect current state

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR updates example1.md to include concrete details of using the edge syncer.

## Related issue(s)

Fixes #

/cc @yana1205 
/cc @dumb0002 
/cc @ezrasilvera 
/cc @waltforme 
@francostellari 
@KPRoche

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
@yuji-watanabe-jp

File: docs/content/en/docs/Coding Milestones/PoC2023q1/example1.md [hunk 1]
```diff
@@ -243,10 +244,179 @@ You can get a listing of those mailbox workspaces as follows.
...
+Do the analogous stuff for the florin cluster.
+
+```console
+$ kubectl ws root:espw
+Current workspace is "root:espw".
+
+$ kubectl ws 4yqm57kx0m6mn76c-mb-bfa7098f-3345-47e6-be45-3e5c5802a234            
+Current workspace is "root:espw:4yqm57kx0m6mn76c-mb-bfa7098f-3345-47e6-be45-3e5c5802a234" (type root:universal).
+
+$ ../kcp/bin/kubectl-kcp workload edge-sync sync-target-f --syncer-image quay.io/kcpedge/syncer:dev-2023-04-18 -o edge-syncer.yaml
```
Comment (yana1205) [inline docs/content/en/docs/Coding Milestones/PoC2023q1/example1.md:386]: I guess this is just a minor mistake.
It's not `../kcp/bin/kubectl-kcp` but `$KCPDIR/bin/kubectl-kcp`

**Comment by MikeSpreitzer** [comment]
The force-push to 6de4d7cd fixes the typo found in review above.

File: docs/content/en/docs/Coding Milestones/PoC2023q1/example1.md (hunk 1 above)
Comment (MikeSpreitzer) [reply docs/content/en/docs/Coding Milestones/PoC2023q1/example1.md]: Right!  Thanks.  Fixed in the force-push to 6de4d7cd

**Comment by yana1205** [comment]
Thank you very much for covering the part of Edge Syncer install in detail.
I went through the step and completed it without any issue

/lgtm

**Pull request merged**